src/platform/tls_openssl.c

```diff
@@ -1964,6 +1972,16 @@ CxPlatTlsProcessData(
                 TlsContext->ResultFlags |= CXPLAT_TLS_RESULT_ERROR;
                 goto Exit;
             }
+        } else if ((TlsContext->SecConfig->Flags & QUIC_CREDENTIAL_FLAG_INDICATE_NULL_CLIENT_CERTIFICATE) &&
+            !TlsContext->PeerCertReceived) {
+            if (!TlsContext->SecConfig->Callbacks.CertificateReceived(
+                TlsContext->Connection,
+                NULL,
+                NULL,
+                0,
+                0)) {
+                // TODO kill the handshake
+            }
```

Comment:
```suggestion
            if (!TlsContext->SecConfig->Callbacks.CertificateReceived(
                  TlsContext->Connection,
                  NULL,
                  NULL,
                  0,
                  0)) {
                TlsContext->ResultFlags |= CXPLAT_TLS_RESULT_ERROR;
                goto Exit;
            }
```
I think this is what you want.